libopenage/pathfinding/integrator.h

```diff
@@ -56,6 +56,7 @@ class Integrator {
 	 * @param other_sector_id Sector ID of the other side of the portal.
 	 * @param portal Portal.
 	 * @param target Coordinates of the target cell, relative to the integration field origin.
+	 * @param time The time to check is the cached cost field is dirty.
```

Comment:
This description is not really useful. How should the user of this method know what time value they have to pass? Caching and dirty checks are also not controlled by the user and therefore don't have to be mentioned.

```suggestion
	 * @param time Time of the path request.
```